.github/workflows/main.yml

```
@@ -6,22 +6,22 @@ jobs:
     name: Run clippy
     steps:
       - name: clippy checkout
```

リポジトリを clone しているだけなので clippy とは関係ないですね
```suggestion
      - name: checkout
```